src/cache/precomputed_quantities.jl

```diff
@@ -17,9 +17,6 @@ Allocates and returns the precomputed quantities:
     - `ᶜp`: the air pressure on cell centers
     - `sfc_conditions`: the conditions at the surface (at the bottom cell faces)
 
-If the `energy_form` is TotalEnergy, there is an additional quantity:
-    - `ᶜh_tot`: the total enthalpy on cell centers
```

Comment:
`- `ᶜh_tot`: the total enthalpy on cell centers`
This line in the doc-string on precomputed quantities is still true in the updated code right ? I'd suggest adding it back in.

Reply:
Right, I will add it back.